docs/source/troubleshooting/general.rst

```diff
@@ -37,3 +37,8 @@ NvFBC, NvENC, or general issues with Nvidia graphics card.
   - You can usually bypass the restriction with a driver patch. See Keylase's
     `Linux <https://github.com/keylase/nvidia-patch>`__
     or `Windows <https://github.com/keylase/nvidia-patch/blob/master/win>`__ patches for more guidance.
+
+
+Packet loss
+-----------
+Some guests might work better with a lower MTU from the host. For example, a LG TV was found to have 30-60% packet loss when the host had MTU set to 1500 and 1472, but 0% packet loss with a MTU of 1428 set in the network card serving the stream (a Linux PC).
```

Comment:
It would probably be good to link the term MTU to https://en.wikipedia.org/wiki/Maximum_transmission_unit for those who aren't networking experts.

Reply:
@skryvel Is that LG TV thing discussed anywhere? (I'm not doubting it, just curious) AFAIK Moonlight requests a certain packet size from Sunshine, based on the MTU it determines (or gets from the OS?). This may of course go wrong, but even then some Moonlight client versions support a `-packet-size` option ([embedded](https://github.com/moonlight-stream/moonlight-embedded/blob/274d3db34da764344a7a402ee74e6080350ac0cd/docs/README.pod#L92), [QT](https://github.com/moonlight-stream/moonlight-qt/blob/9117f6565e4b2a6ba5417282de6bf9360b681f1a/app/cli/commandlineparser.cpp#L431)) in order to override. Wouldn't that achieve the same result?

I'm not sure a blanket recommendation to mess with the MTU is warranted. Maybe only if specific clients are used that don't have such an option?

Reply:
@gschintgen no and that reminded me to SSH in and look around. In the TV the `eth0` interface has MTU of `1500` and I can get `ping` to work with packets of up to `1492` (including headers) so I don't know. I'll word it out so it's framed as a "last resort" idea but the diagnosing was me altering MTU on my host and witnessing the changes in the stream instantly, after trying everything else. Also of note, while local Sunshine streaming had terrible packet loss, Xbox cloud was streaming fine IIRC. Maybe my router or some multi-device interaction ? I really don't know.